<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reservationGui</class>
 <widget class="QDialog" name="reservationGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="selectedRoomLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected room: </string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>381</width>
     <height>187</height>
    </rect>
   </property>
   <property name="selectedDate">
    <date>
     <year>2025</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SelectionMode::NoSelection</enum>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>110</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>610</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>End date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="endDate">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>110</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="reserveRoom">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reserve</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorStartDate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: red;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorEndDate">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: red;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
